requirements.txt

```diff
@@ -1,8 +1,9 @@
-stackvana
+stackvana<=0.2023.28
```

Comment:
We should discuss this.  Were you having difficulty with new versions?

Reply:
hmm, at some point it has an error but I forgot the details. Let me test this part...